### ISSUE TITLE
Microsoft Edge File object missing lastModified

### ISSUE THREAD

**Issue by Slike9** [therealparmesh/object-to-formdata#13]
In version 1.3.0 File objects are converted to FormData improperly in MS Edge.

Recently File checking has been changed to (in https://github.com/therealparmesh/object-to-formdata/commit/bd1b2c6a46da594929f546e7595664631477be37):
```js
 function isFile (value) {
-  return value instanceof File
+  return isBlob(value) &&
+      typeof value.lastModified === 'number' &&
+      typeof value.name === 'string'
 }
```
But in MS Edge File object doesn't have lastModified property https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/4155006/, instead it has lastModifiedDate of 'object' type.

**Comment by piranna** [comment]
According to spec, `lastModifiedDate` is deprecated in benefict of `lastModified`. This code could also get deprecated soon, because FormData allows to use Blob object instead so checking will be done against it, so when https://github.com/therealparmesh/object-to-formdata/pull/12 get merged `lastModified` will not be used anymore.

**Comment by huongquoine** [comment]
Thank @Slike9 for reporting this issue. `object-to-formdata` is not working on Microsoft Edge cause the issue.

Why don't you use `value instanceof File` anymore, @therealparmesh?

**Comment by therealparmesh** [comment]
@huongquoine I believe we made a change to use duck-typing to check for `File` and `Blob` so that we could support server-side in addition to client-side.

@piranna I'm not sure how I feel about using detection. I'm thinking that we should just do some tactical `||`s to re-add support for Edge in the meantime.

**Comment by piranna** [comment]
> @piranna I'm not sure how I feel about using detection. I'm thinking that we should just do some tactical ||s to re-add support for Edge in the meantime.

I agree on that, it would not hurt. On the other hand, since FormData allow to use Blob objects, would be good to check if Edge support them too, so we would not need to worry about Edge or File objects anymore...

**Comment by therealparmesh** [comment]
The fix is in. Thank you!
